.github/workflows/build.yml

```diff
@@ -108,7 +108,7 @@ jobs:
       message_file: ${{ steps.pr_comment.outputs.message-file }}
       webCatalogOnly: ${{ steps.check_pr_content.outputs.webCatalogOnly }}
       chartEntryName: ${{ steps.check_pr_content.outputs.chart-entry-name }}
-      chartNameWithVersion: ${{ steps.check_pr_content.outputs.chart-name-with-version }}
```

Comment:
Contextually, I do feel like "chart name with version" is a bit more clear. It's descriptive of the format of the data without showing the data.

With that said, I'd consider something like `chart_release_tag` to be acceptable. It's still descriptive, if only slight less so in terms of the content, but moreso in terms of its use.

Otherwise, I think I prefer it the way it is.

@jsm84 is out at the moment but if this is still open when they return, they can chime in.
@OchiengEd please feel free to add your thoughts as well.

Reply:
I like how `chartNameWithVersion` is descriptive; you can picture the output without seeing its contents. Not opposed to renaming, but would recommend making it descriptive.

Lastly, all other variables are now using, snake case so the rename might help with bringing in consistency.

Reply:
I would rather describe the usage rather than the content. `release_tag` describes the GitHub Tag that is used to cut the release of this chart. This variable / output parameter is only used for this purpose. (I wouldn't argue if we'd use the `chartNameWithVersion` in some other place for some other purpose that the GitHub release tag).

Also, `chartNameWithVersion` is actually not an completely accurate description of the format of the data. Its actually formatted as `tag_name = f"{organization}-{chart}-{version}"` so for instance `hashicorp-vault-0.25.0`. So maybe it should be `organizationWithChartNameAndVersion` ? :)

Naming is hard ...

Reply:
I suppose since it is only being used in a single location, we can afford to rename to `release_tag`.

Reply:
I'll resolve the conversation. Thanks and sorry if that feels like bikeshedding !

(@komish It also makes sense how you see it and explain it)